### ISSUE TITLE
Topic search

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#55]
It would be nice to have a page from which the user can search all available topics that they can play. This needs to be both a local and remote search since the user may have lessons downloaded, but no internet connectivity. Or, conversely, the user may have internet connectivity and no lessons downloaded.

**Comment by BenHenning** [comment]
This is blocked on #22 for a place from which to open the topic search page, #15 and #12 for local searching, and #5 for the core architecture of endpoints that needs to be extended to include topic searching. This may also require a utility for checking whether the user has internet access--this is likely infrastructure work that will be completed as part of earlier projects. This also may require Oppia backend work to introduce topic searching.

**Comment by BenHenning** [comment]
At this point, it doesn't seem particularly important to support this since the number of topics is small. We can revisit this as a feature once there's an observed error to solve.
